apps/ios/src/index.tsx

```diff
@@ -4,13 +4,13 @@ import { customRegistry, FabricTester, IFabricTesterProps } from '@fluentui-reac
 import { ThemeProvider } from '@uifabricshared/theming-react-native';
 import * as React from 'react';
 import { AppRegistry, SafeAreaView } from 'react-native';
-import { allPlatformTests } from '../../tests/AllPlatformTests';
+import { iosTests } from './IOSTests';
 
```

Comment:
For my own sake, what was wrong with allPlatformTests that we need to fall back to iOS?

Reply:
Nevermind, you answered this elsewhere